bears/general/SpaceConsistencyBear.py

```diff
@@ -41,11 +42,21 @@ def run(self,
                 if replacement[-1] != "\n":
                     replacement += "\n"
                     result_texts.append("No newline at EOF.")
+                    additional_info_texts.append(
+                        "A trailing newline character ('\\n') is missing from "
+                        "your file. "
+                        "<http://stackoverflow.com/a/5813359/3212182> will "
+                        "give you more information about why this may or may "
+                        "not be important to you.")
```

Comment:
too long here i think. what about ``gives you more information on why you would need one.``

- change to present tense
- it's a little shorter